test(e2e): fix websocket connecting when using multiple http test servers

Currently, the E2E tests cause the following panic when all tests are ran with `go test -race -cover`:
```bash
panic: repeated read on failed websocket connection

goroutine 24 [running]:
github.com/gorilla/websocket.(*Conn).NextReader(0xc0000ea580, 0xffffffffffffffff, 0x0, 0xc000257fa0, 0x493df0, 0xc000257e80)
	/home/carson/go/pkg/mod/github.com/gorilla/websocket@v1.4.2/conn.go:1001 +0x603
github.com/gorilla/websocket.(*Conn).ReadJSON(0xc0000ea580, 0x8f3f80, 0xc000332420, 0x1, 0x0)
	/home/carson/go/pkg/mod/github.com/gorilla/websocket@v1.4.2/json.go:50 +0x3d
github.com/zaba505/graphql-transport-ws.(*Conn).readMessages(0xc0000908c0)
	/home/carson/Desktop/Go/graphql-transport-ws/conn.go:46 +0x17a
created by github.com/zaba505/graphql-transport-ws.newConn
	/home/carson/Desktop/Go/graphql-transport-ws/conn.go:35 +0x1c3
exit status 2
FAIL	github.com/zaba505/graphql-transport-ws	0.019s
```

My suspicion is that this is coming from simultaneous `httptest.NewServer` calls.

--- a/conn.go
+++ b/conn.go
@@ -44,7 +44,7 @@ func (c *Conn) readMessages() {
 	for {
 		var msg operationMessage
 		err := c.wc.ReadJSON(&msg)
-		if err != nil && websocket.IsUnexpectedCloseError(err, websocket.CloseNormalClosure, websocket.CloseAbnormalClosure) {
+		if err != nil {
 			return
 		}
 
@@ -64,7 +64,7 @@ func (c *Conn) writeMessages() {
 			return
 		case op := <-c.out:
 			err := c.wc.WriteJSON(&op)
-			if err != nil && websocket.IsUnexpectedCloseError(err, websocket.CloseNormalClosure, websocket.CloseAbnormalClosure) {
+			if err != nil {
 				return
 			}
 		}

